cookbooks/adiabatic_boundary.prm

```diff
@@ -0,0 +1,80 @@
+set Dimension                              = 3
+
+set Use years in output instead of seconds = true
+set End time                               = 0
+set Nonlinear solver scheme                = single Advection, single Stokes
+set Nonlinear solver tolerance             = 1e-5
```

Comment:
You can remove the line with `set Nonlinear solver tolerance` since the model does not do non-linear iterations.